package.json

```diff
@@ -28,10 +28,12 @@
     "@emotion/styled": "^10.0.0",
     "babel-plugin-emotion": "^10.0.0",
     "emotion-theming": "^10.0.0",
+    "moment": "^2.24.0",
```

Comment:
I don't know if there's a better way to add this, since it's just a dependency of react-datetime.

Reply:
I think this is right. I came across this library recently and they tell you to do it if you want to use certain features. They should probably include it as a dependency.